crates/dojo-lang/src/manifest_test_data/compiler_cairo_v240/Scarb.lock

```diff
@@ -10,7 +10,7 @@ dependencies = [
 
 [[package]]
 name = "dojo"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
i think this should be reverted since we haven't released 0.4.2. i accidentally versioned the cargo previously

Reply:
Totally, thank you for pointing this out.